plugins/CircuitViewer/tests/braynsTestData.cpp

```diff
@@ -29,12 +29,35 @@
 #include <brayns/common/renderer/FrameBuffer.h>
 #include <brayns/common/scene/Model.h>
 #include <brayns/common/scene/Scene.h>
+#include <brayns/common/simulation/AbstractSimulationHandler.h>
 
 #include <BBP/TestDatasets.h>
 
 #define BOOST_TEST_MODULE braynsTestData
 #include <boost/test/unit_test.hpp>
 
+#include <chrono>
+#include <thread>
+
+namespace
+{
+constexpr size_t WAIT_MILLISECONDS = 5000;
+const auto waitForSimulationUpload = [&](const auto& simulationHandler) {
```

Comment:
Did you forget how to declare a normal function? :)

Reply:
Yes :) this is now removed.